src/components/shadows/examples/shadow-examples.scss

```diff
@@ -16,7 +16,8 @@
 
     code {
         font-size: pxToRem(12);
-        background-color: rgb(var(--contrast-500));
+        background-color: rgb(var(--kompendium-contrast-500));
+        color: rgb(var(--kompendium-contrast-1100));
```

Comment:
I'll retarget this onto the `upgrade-Kompendium-from-v.0.7.3-to-v.0.8.5` branch, because without the updated Kompendium, the checkboxes in the _Color system_ guide look like this:

![image](https://user-images.githubusercontent.com/154725/118807129-3188d680-b8a8-11eb-94a4-e8d1c19c2298.png)